scrollinga.js

```diff
@@ -127,7 +127,11 @@ class Scrollinga {
    * @return {Boolean}
    */
   isScrollDown() {
-    return (this.target.scrollTop + this.target.clientHeight === this.target.scrollHeight);
+    const pixelRatio = window.devicePixelRatio || 1;
+    const client = Math.floor((this.target.clientHeight + this.target.scrollTop) / pixelRatio);
+    const scroll = Math.floor(this.target.scrollHeight / pixelRatio);
+
+    return client === scroll;
```

Comment:
Maybe better names like `clientHeight` and `scrollHeight` to map original not pixelated variables